app/src/pages/inside/logsPage/logToolbar/logToolbar.scss

```diff
@@ -18,3 +18,31 @@
   display: inline-block;
   margin-right: -1px;
 }
+
+@media (min-width: $SCREEN_SM_MIN) {
+  .action-button:first-child {
+    padding-left: 53px;
+  }
+}
+@media (max-width: $SCREEN_XS_MAX_DASHBOARD) {
+  .log-toolbar {
+    flex-direction: column;
+    align-items: flex-start;
+    border-bottom: none;
+  }
+
+  .action-buttons {
+    align-self: flex-end;
+    margin-top: 8px;
+  }
+}
+
+.action-button:nth-child(2) {
+  padding-left: 0px;
+}
+
+@media (min-width: 954px) {
```

Comment:
Please, follow comment form the [link](https://github.com/reportportal/service-ui/pull/1071#discussion_r248955146)